### PR TITLE
feat: redeploy spoke pools to support depositExclusive

### PR DESCRIPTION
**Pull request by bmzig**
This redeploys all spoke pools which were deployed before Lisk to support `depositExclusive`. A nice byproduct is that now all chains also support tryMulticall.

### CODE REVIEW

**Review by james-a-morris — APPROVED** [review]
lgtm pending at least a second signoff

File: hardhat.config.ts
Comment (nicholaspai) [inline hardhat.config.ts:139]: these were renamed by infura?

Comment (bmzig) [reply hardhat.config.ts:139]: Yeah they must have been. They now follow a naming convention given by `[network_name]-[mainnet or sepolia].infura.io.`

Comment (nicholaspai) [reply hardhat.config.ts:139]: huh ok that makes sense, does this just not work currently with the renamed network names?

**Review by nicholaspai — APPROVED** [review]

**Pull request merged**
